bciers/apps/administration-e2e/tests/workflows/industry_user_operator/operator-with-beforeAll.spec.ts

```diff
@@ -0,0 +1,107 @@
+// 🧪 Suite to test the administration "industry_user_operator" (user with access to an operator) workflow
+// tests that need fixture setup only once
+import { test } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/administration-e2e/poms/dashboard";
+import { OperatorPOM } from "@/administration-e2e/poms/operator";
+// ☰ Enums
+import { AppRoute } from "@/administration-e2e/utils/enums";
+import { AppName } from "@/administration-e2e/utils/constants";
+import { UserRole } from "@bciers/e2e/utils/enums";
+// 🛠️ Helpers
+import {
+  analyzeAccessibility,
+  setupTestEnvironment,
+} from "@bciers/e2e/utils/helpers";
+const happoPlaywright = require("happo-playwright");
+
+test.beforeAll(async () => {
+  // Note: can run multiple times if using multiple workers (or, if a test fails you'll get a new worker- can't be helped)
+  // So, ensure this runs only once by using only 1 worker
+  // Setup fixtures for admin-industry_user
+  await setupTestEnvironment(AppName + UserRole.INDUSTRY_USER + "_operator");
```

Comment:
`await setupTestEnvironment(AppName + "-" + UserRole.INDUSTRY_USER + "_operator");`

Reply:
Thank you for noticing this issue 🙏 -  it's a big one, my bad... 

-10 points from Ravenclaw, +10 points to Gryffindor!

Aside:
I think the typo went un-noticed in setting up the right environment using setupTestEnvironment because  `bciers/apps/administration-e2e/tests/workflows/industry_user/select-operator-with-beforeEach.spec.ts` tests ran before `bciers/administration-e2e/tests/workflows/industry_user_operator/operator-with-beforeAll.spec.ts` essentially creating the approved operator access needed for role `industry_user` tests for"My operator".